packages/web-pkg/src/components/FilesList/ResourceTile.vue

```diff
@@ -130,6 +137,10 @@ defineSlots<{
   selection?: (props: { item: Resource }) => unknown
 }>()
 
+function toggleTile(data: [Resource, MouseEvent | KeyboardEvent], event?: MouseEvent) {
```

Comment:
Just for consistency inside setup functions:

```suggestion
const toggleTile = (data: [Resource, MouseEvent | KeyboardEvent], event?: MouseEvent) => {
```